commit 823e1331db5b085f4a86d6f8d457612edc459c12
Author:     Woo Chang Kim <wchang.kim@samsung.com>
AuthorDate: Thu Oct 22 06:49:16 2026 +0900
Commit:     Woo Chang Kim <wchang.kim@samsung.com>
CommitDate: Thu Oct 22 06:49:16 2026 +0900

    Change the repo path of Tizen-Base-X in snapshot_info
    
    Change-Id: I33d440d54974728ba4ec4116cf8d2cc3df2ba096
---
 TIZEN/Tizen/Tizen-Base-X/snapshot_info | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/TIZEN/Tizen/Tizen-Base-X/snapshot_info b/TIZEN/Tizen/Tizen-Base-X/snapshot_info
index 9576d4a..ba761dd 100644
--- a/TIZEN/Tizen/Tizen-Base-X/snapshot_info
+++ b/TIZEN/Tizen/Tizen-Base-X/snapshot_info
@@ -1,8 +1,8 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <snapshotinfo>
-  <latest target="tizen-base-x_20261020.230437" url="http://download.tizen.org/snapshots/TIZEN/Tizen/Tizen-Base-Dev/"/>
-  <reference target="tizen-base-x_20261020.230437" url="http://download.tizen.org/snapshots/TIZEN/Tizen/Tizen-Base-Dev/"/>
-  <base target="reference" url="http://download.tizen.org/snapshots/TIZEN/Tizen/Tizen-Base-Dev/">
+  <latest target="tizen-base-x_20261020.230437" url="http://download.tizen.org/snapshots/TIZEN/Tizen/Tizen-Base-X/"/>
+  <reference target="tizen-base-x_20261020.230437" url="http://download.tizen.org/snapshots/TIZEN/Tizen/Tizen-Base-X/"/>
+  <base target="reference" url="http://download.tizen.org/snapshots/TIZEN/Tizen/Tizen-Base/">
     <repository name="standard">
       <type name="standard"/>
     </repository>
